File: nan21.dnet.module.mm/nan21.dnet.module.mm.ui.extjs/src/main/webapp/src/ds/SubInventoryLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.mm.inventory.ds.model.SubInventoryLovDs" , {
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id",type:"int", useNull:true },
		{name:"clientId",type:"int", useNull:true },
		{name:"name",type:"string" },
		{name:"active",type:"boolean" },
		{name:"inventoryId",type:"int", useNull:true }
	]
});
	
Ext.define("net.nan21.dnet.module.mm.inventory.ds.param.SubInventoryLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.md/nan21.dnet.module.md.presenter/src/main/java/net/nan21/dnet/module/md/org/ds/model/SubInventoryLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2010 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.md.org.ds.model;

import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;

import net.nan21.dnet.module.md.org.domain.entity.SubInventory;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = SubInventory.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = SubInventoryLovDs.fNAME) })
public class SubInventoryLovDs extends AbstractTypeLov<SubInventory> {

    public static final String fINVENTORYID = "inventoryId";

    @DsField(join = "left", path = "inventory.id")
    private Long inventoryId;

    public SubInventoryLovDs() {
        super();
    }

    public SubInventoryLovDs(SubInventory e) {
        super(e);
    }

    public Long getInventoryId() {
        return this.inventoryId;
    }

    public void setInventoryId(Long inventoryId) {
        this.inventoryId = inventoryId;
    }

}
